packages/@react-aria/grid/src/GridKeyboardDelegate.ts

```diff
@@ -22,7 +22,8 @@ export interface GridKeyboardDelegateOptions<T, C> {
   direction: Direction,
   collator?: Intl.Collator,
   layout?: Layout<Node<T>>,
-  focusMode?: 'row' | 'cell'
+  focusMode?: 'row' | 'cell',
+  cycleMode?: 'none' | 'within' | 'between'
```

Comment:
I'm not entirely sure what `cycleMode` is supposed to be here, is it to control focus behavior within a grid cell/row/collection and make it wrap back to the start? Lemme ask the team to see if anyone can describe the behavior we want here for the Tag items.